docs/examples/general/data_loading/dataloading_webdataset.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<p><em>I think it reads slightly better without "that is", which introduces another verb into the sentence:</em></p><p>This example shows you how to use the data <s>that is</s> stored in <s>the</s> Webdataset format with DALI. </p><p> </p><p>Creating <strong>an</strong> index</p><p> </p><p>I think that this paragraph misses the point - it's more about "Using readers.webdataset operator" and not about "creating an index".</p><p>I'd suggest splitting it like this:</p><p> </p><h2>Using readers.webdataset operator</h2><p> </p><p><s>To use data that is stored in the Webdataset format, we need to use the </s><code><s>readers.webdataset</s></code><s> operator </s><strong>Data stored in WebDataset format can be read with </strong><code><strong>readers.webdataset</strong></code><strong> operator.</strong></p><p><s>In addition to the arguments that are common to all readers, such as </s><code><s>random_shuffle</s></code><s>, this operator takes </s><code><s>paths</s></code><s>, </s><code><s>index_paths</s></code><s>, </s><code><s>ext</s></code><s>, </s><code><s>missing_component_behavior</s></code><s> and </s><code><s>dtypes</s></code><s> arguments: </s></p><p>This operator takes the following arguments:</p><ul><li><code>paths</code> the path (or list of paths) to the tar archives containing <s>webdataset</s> the dataset</li><li><code>index_paths</code> the path (or list of paths) to the respective index files containing data about the specifics of the tar files.<s> Each index file only has to be generated once and can be created using the </s><code><s>wds2idx</s></code><s> </s>utility included with DALI. For the <s>specifics of the </s>usage <strong>details, </strong>please refer to <code>wds2idx -h</code></li><li><code>ext</code> the extension set (or list of those) of extensions separated by a ';' that specify the outputs of the operator and which sample components will be fed into the specific outputs of the operator</li><li><code>missing_component_behavior</code> the behavior of the reader in case it encounters a sample that can't return provide any component to a certain output. There are 3 options:</li><li class="ql-indent-1"><code>empty</code> (default) <s>just </s>returns an empty tensor for that output</li><li class="ql-indent-1"><code>skip</code> the sample<strong>s with missing components are</strong> <s>will be</s> skipped<s> and the next valid one will be taken</s></li><li class="ql-indent-1"><code>error</code> an error <s>will be</s> <strong>is</strong> raised</li><li><code>dtypes</code> the <s>dtypes</s> <strong>data types </strong>of the outputs of the operator. If the size of the output component is not divisible by the size of the type, an error <s>will be thrown</s><strong> is raised</strong>. <s>INT8 </s><strong><s>I</s>f</strong> not provided, <strong>data is returned as UINT8</strong></li></ul><p> </p><p><strong>In addition to these arguments, the operator accepts arguments common to all readers, which configure the seed of random number generator, shuffling, sharding and handling of incomplete batches at the end of epoch.</strong></p><p> </p><h2>Creating an index</h2><p> </p><p>The index file(s), paths to which are passed in the argument <code>index_paths</code> , can be generated with a tool <code>wds2idx</code>,  bundled with DALI.</p><p> </p><p><strong>Note:</strong> To run the code below, make sure that <code>DALI_EXTRA_PATH</code> environment variable points to the location where data from <a href="https://github.com/NVIDIA/DALI_extra" target="_blank">DALI extra repository</a> is downloaded and that it's checked out at the tag matching your version of DALI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I don't think we need to repeat the `readers.webdataset` documentation here. I would just ask the user to check it for the details.

Reply:
<p>that's how other reader guides do that</p>

Reply:
<p>ok</p>

Reply:
<p>JanuszL: The text now is a copy-paste from my suggestion, so I'm relying on your approval here.</p>

Reply:
<p><code>The index file(s), paths to which are passed in the argument index_paths , can be generated with a tool wds2idx, bundled with DALI.</code></p><p> -&gt;</p><p><code>The index file(s), paths to which are passed in the argument index_paths, can be generated with a tool wds2idx, bundled with DALI.</code></p>